core/src/test/java/org/commonjava/maven/ext/core/fixture/TestUtils.java

```diff
@@ -90,24 +91,38 @@ public static Model getDummyModel ()
         return result;
     }
 
+    // Public API for manipulator-examples-project
+    public static ManipulationSession createSession( Properties p, File pom ) throws ManipulationException
+    {
+        return createSessionAndManager( p, pom ).getSession();
+    }
```

Comment:
Why is this been created?

Reply:
Per our discussion, since you want to use `session.getPom()` instead of using the project, then the pom `File` must be passed into the session. Currently, sessions are being created with a `null` pom in `createSession`. You had said to fix the tests rather than have that `if` statement so that's why I added a method to take a `File`.

Reply:
Ah right, the `Public API for manipulator-examples-project` confused me. Can you do me a favour and layer the commits as its impossible to see the changes between subsequent pushes and I have to re-review everything again - which is rather time consuming. You can also squash at the end once its complete.

Reply:
Can we alter the comment please

Reply:
I am not sure yet if this change breaks manipulator-examples-project and then we'd have to switch to the new API.

Reply:
You mean whether manip-examples needs to use this API?